src/JestTestRunner.ts

```diff
@@ -33,6 +42,14 @@ export default class JestTestRunner extends EventEmitter implements TestRunner {
     };
   }
 
+  private setNodeEnv() {
+    // Jest CLI will set process.env.NODE_ENV to 'test' when it's null, do the same here
+    // https://github.com/facebook/jest/blob/master/packages/jest-cli/bin/jest.js#L12-L14
+    if (this.processEnvRef.NODE_ENV == null) {
```

Comment:
Why not use a triple check?

Reply:
`undefined` also need to

Reply:
`jest-cli` does not use triple equal check, `undefined` need to be involved in comparison too

Reply:
Ye, are we going the same route as jest or is `if(this.processEnvRef.NODE_ENV)` sufficient? I don't see a situation in which they're not equivalent.